frontend/src/components/SignIn/SignIn.js

```diff
@@ -12,6 +12,7 @@ class SignIn extends Component {
       isPasswordVisible: false,
       email: '',
       password: '',
+      error: '',
```

Comment:
Good job adding in an empty string for error and updating the state!

Reply:
Can we change the name of this variable to something like `validationError` or `validationErrorText`? That way it'll be a little easier to understand/distinguish from other things called `error`?

Reply:
Yeah Bryan..will worked on it.